pyproject.toml

```diff
@@ -1,28 +1,36 @@
-[build-system]
-requires = ["setuptools>=61.0"]
-build-backend = "setuptools.build_meta"
-
-[project]
-name = "arcaflow_plugin_sdk"
+[tool.poetry]
+name = "arcaflow-plugin-sdk-python"
 # Do not change, the version is automatically updated in CI.
 version = "0.0.0"
-authors = [
-]
 description = "Plugin SDK for the Arcaflow workflow engine"
+authors = ["Arcalot Contributors"]
+license = "Apache-2.0"
 readme = "README.md"
-license = { text="Apache-2.0" }
-requires-python = ">=3.9"
+packages = [{include = "arcaflow_plugin_sdk", from = "src"}]
```

Comment:
The other projects depend on `arcaflow-plugin-sdk`, not `arcaflow-plugin-sdk-python`, so this would be a breaking change. 
The python is not required, since it's assumed that it's Python when using python.

After this is merging, it's ready for merging. I tested it on a plugin, and this was the only problem.

Reply:
Corrected.